server/lib/user/controller.ts

```diff
@@ -11,4 +11,31 @@ export async function getUser(event: APIGatewayEvent, _: Context, callback: APIG
   }
 
   callback(null, { statusCode: 200, body: JSON.stringify({ data: '' }) });
+}
+
+export async function saveUser(event: APIGatewayEvent, _: Context, callback: APIGatewayProxyCallback) {
```

Comment:
All this should be done in separate file, as example, create new folder, name it save-user, then create file named controller.ts and paste the code.